lib/chef/client.rb

```diff
@@ -322,22 +294,11 @@ def run
         runlock.release
       end
 
-      # Raise audit, converge, and other errors here so that we exit
+      # Raise converge, and other errors here so that we exit
       # with the proper exit status code and everything gets raised
       # as a RunFailedWrappingError
-      if run_error || converge_error || audit_error
-        error = if Chef::Config[:audit_mode] == :disabled
-                  run_error || converge_error
-                else
-                  e = if run_error == converge_error
-                        Chef::Exceptions::RunFailedWrappingError.new(converge_error, audit_error)
-                      else
-                        Chef::Exceptions::RunFailedWrappingError.new(run_error, converge_error, audit_error)
-                      end
-                  e.fill_backtrace
-                  e
-                end
```

Comment:
i think you still need this bit of code or at least something like:

```
      if run_error || converge_error 
        error =  if run_error == converge_error
                        Chef::Exceptions::RunFailedWrappingError.new(converge_error)
                      else
                        Chef::Exceptions::RunFailedWrappingError.new(run_error, converge_error) 
                      end
                  error.fill_backtrace
```

Reply:
oh maybe nevermind though, i just read through this with my last comment in mind and yeah it looks like this all goes away because `:audit_mode == :disabled` now.